### PR TITLE
snapper: replace file existance check with file content checksum check

### PR DESCRIPTION
**Pull request by asmorodskyi**
replace file existance check with file content checksum check

### CODE REVIEW

**Comment by asmorodskyi** [comment]
VR: http://autobot.suse.de/tests/647

**Comment by asmorodskyi** [comment]
VR: http://autobot.suse.de/tests/655

File: tests/console/snapper_jeos_cli.pm [hunk 1]
```diff
@@ -34,15 +34,17 @@ sub run {
...
+    my $openqalatest  = script_output("snapper create -p -d openqalatest");
     assert_script_run("snapper list");
 
     $self->rollback_and_reboot($openqainit);
     assert_script_run("! ls -l $file");
 
     $self->rollback_and_reboot($openqalatest);
-    assert_script_run("ls -l $file");
+    my $checksum = script_output("sha256sum $file");
+    die("Restored file sha256 checksum didn't match") unless $checksum_orig eq $checksum;
```
Comment (mloviska) [inline tests/console/snapper_jeos_cli.pm:47]: Neat picking and sugar syntax warning! :D
```
... if $checksum_orig ne $checksum;
```

Comment (asmorodskyi) [reply tests/console/snapper_jeos_cli.pm:47]: actually I would agree , I have read advice somewhere that better do "positive coding" meaning exactly what you suggesting here

**Comment by asmorodskyi** [comment]
Done , also I restart a job to see if it still works ( but should obviously work ) 
VR: http://autobot.suse.de/tests/656

**Pull request merged**